src/bootstrap.ts

```diff
@@ -24,17 +31,20 @@ let loopStart: number = Date.now();
  * Watch mode should/can be reliably left running for weeks/months as CouchDB is made for that.
  * BUT for the moment it's mandatory to relaunch it because it's the only way to update: typescript, downloads stats.
  */
-async function run(
+export async function run(
   stateManager: StateManager,
   algoliaClient: SearchClient,
   mainIndex: SearchIndex,
   bootstrapIndex: SearchIndex
 ): Promise<void> {
+  log.info('-----');
+  log.info('⛷   Bootstrap: starting');
```

Comment:
in this PR you remove some of the log emojis (like in run), should they all be removed as it's not consistent anymore, added back, or is it fine with a mix?

Reply:
I removed some to focus on readability, just leaving on the main logs. I'm good with a mix, it highlight big steps